foia_hub/settings/local_settings.py

```diff
@@ -1,13 +1,19 @@
 from .base import *
 import os
+import re
 import dj_database_url
+from cfenv import AppEnv
+
+database_url = os.getenv("DATABASE_URL")
+
+env = AppEnv()
+cf_foia_db = env.get_service(name=re.compile('foia-db'))
+if cf_foia_db:
+    database_url = cf_foia_db.credentials['uri']
 
 # See env.example for an explanation of these settings.
 
 SECRET_KEY = os.getenv("FOIA_SECRET_SESSION_KEY")
-DATABASES = {'default': dj_database_url.parse(os.getenv("DATABASE_URL"))}
+DATABASES = {'default': dj_database_url.parse(database_url)}
```

Comment:
Heads up, this can be `dj_database_url.config()`, which will read from `os.getenv['DATABASE_URL']`.